### PR TITLE
Fix: Messenger handle failed messenger token

### PR DESCRIPTION
**Pull request by maxklenk**
Pass error from the SC-server to the client.

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5325647&bid=17430235)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5325647&bid=17430235)

File: controllers/messenger.js [hunk 1]
```diff
@@ -8,6 +8,13 @@ router.use(authHelper.authChecker);
 
 router.get('/token', (req, res, next) => api(req)
 	.post('/messengerToken', {})
-	.then((result) => res.send(result)));
+	.then((result) => res.send(result))
+	.catch((e) => {
+		// eslint-disable-next-line no-console
+		console.error('Failed to get messengerToken', e.error);
+		const error = new Error('Messenger Token nicht verfügbar.');
```
Comment (tofixx) [inline controllers/messenger.js:15]: use `catch(next)`

File: controllers/messenger.js [hunk 2]
```diff
@@ -8,6 +8,13 @@ router.use(authHelper.authChecker);
 
 router.get('/token', (req, res, next) => api(req)
 	.post('/messengerToken', {})
-	.then((result) => res.send(result)));
+	.then((result) => res.send(result))
+	.catch((e) => {
+		// eslint-disable-next-line no-console
+		console.error('Failed to get messengerToken', e.error);
```
Comment (tofixx) [inline controllers/messenger.js:14]: use logger instead, no console!

File: .eslintrc.json [hunk 3]
```diff
@@ -5,6 +5,13 @@
 		"plugin:json/recommended"
 	],
 	"rules": {
+		"arrow-parens": [
```
Comment (tofixx) [inline .eslintrc.json:8]: this is not part of the pr
